src/ray/common/ray_config_def.h

```diff
@@ -34,6 +34,9 @@ RAY_CONFIG(uint64_t, num_heartbeats_warning, 5)
 /// The duration between dumping debug info to logs, or -1 to disable.
 RAY_CONFIG(int64_t, debug_dump_period_milliseconds, 10000)
 
+/// Whether to enable fair queueing between task classes in raylet.
```

Comment:
It would be good to have more description of what "fair queuing" and "task classes" mean since this is where developers will probably look first.